bin/cliain/src/staking.rs

```diff
@@ -38,5 +38,5 @@ pub fn set_staking_limits_command(
 }
 
 pub fn force_new_era_command(root_connection: Connection) {
```

Comment:
this function seems to be just adding unnecessary indirection?

Reply:
Right, it was done (by me) to comply with `*_command` function name convention but I guess we don't need that additional level of indirection. I can volunter to remove it in my next PR.

Reply:
there was an issue raised today by @h4nsu that actually `cliain` should be just a binary interface for `aleph-client`

Reply:
Yes, we can back up this and push this idea of binary interface. Btw how does exactly the solution looks like? We'd have one package `aleph-client` which will have `lib.rs` and `main.rs`? This will work, however one would not be able to depend on that package since that package would be a binary one.

Reply:
Ok we spoke on the side and turns out we can make `cliain` and `aleph-node` distinct rust crates in one `Cargo.toml` in the same folder on FS. That would work for me.